doc/user/layouts/partials/create-source/envelope/debezium/details.html

```diff
@@ -62,13 +62,9 @@
 
 #### Kafka topic requirements
 
-Every Debezium message provides enough information to transition betwen the previous state of one
-row of a table to the immediate successor state of that row. If you enable [log compaction][] on
-Kafka topics containing Debezium messages then Kafka may erase an arbitrary prefix of changes to
-rows.
-
-In order for Materialize to handle compacted topic you must specify that data is in the `DEBEZIUM
-UPSERT` format, in which case Materialize will retain enough information about seen rows to
-handle arbitrary missing change log prefixes.
+`ENVELOPE DEBEZIUM` by itself is incompatible with Kafka's [log compaction][]. If you enable
+compaction of a topic carrying Debezium data you must specify `ENVELOPE DEBEZIUM UPSERT`. The
```

Comment:
Insert comma after "Debezium data".

Reply:
I restructured the sentence to make the comma unnecessary (IMO).